src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -87,4 +87,11 @@ public void addUser(@RequestParam(name = "userId") UUID userId, @PathVariable(na
     public List<Skill> getSkillList (){
         return Arrays.asList(Skill.values());
     }
+
+    @SuperAdminAccessControl
+    @PutMapping("/{internshipId}/update")
```

Comment:
Why do you need `update` word?
```suggestion
    @PutMapping("/{internshipId}")
```

Reply:
done